withDom.js

```diff
@@ -1,3 +1,5 @@
+require('raf/polyfill');
```

Comment:
Isn't that kind of the point of `withDom` though? It's responsible for setting up global `document`, `window`, etc. I don't see how it would work if it didn't modify globals.
```javascript
    global.document = jsdom('');
    global.window = document.defaultView;
```